### PR TITLE
Also use rel-eng scripts to create the release tags

### PR DESCRIPTION
**Pull request by ekohl**
This further automates the jobs with more scripts. This is incomplete, but comes from https://github.com/theforeman/tool_belt/pull/378#discussion_r653858611. It's separated to its own PR so that PR wasn't held up.

### CODE REVIEW

File: procedures/foreman/release.md.erb [hunk 1]
```diff
@@ -42,22 +42,18 @@
...
-- In foreman-selinux <%= short_version %>-stable:
-  - [ ] Tag the release using [tag.sh](https://gist.github.com/tbrisker/d51cedea51ccbcca00e72c1fc61550fe) `tag.sh <%= full_version %> && git push upstream <%= short_version %>-stable --follow-tags`
-- In foreman-installer <%= short_version %>-stable:
-  - [ ] Tag the release using [tag.sh](https://gist.github.com/tbrisker/d51cedea51ccbcca00e72c1fc61550fe) `tag.sh <%= full_version %> && git push upstream <%= short_version %>-stable --follow-tags`
-- [ ] Run the Jenkins [Tarballs Release](https://ci.theforeman.org/job/tarballs-release/) to create tarballs
+- [ ] Make sure [test_<%= short_version.tr('.', '_') %>_stable](https://ci.theforeman.org/job/test_<%= short_version.tr('.', '_') %>_stable/) and [test_<%= short_version.tr('.', '_') %>_stable](https://ci.theforeman.org/job/test_proxy_<%= short_version.tr('.', '_') %>_stable/) are green using [verify_green_jobs](https://github.com/theforeman/theforeman-rel-eng/blob/master/verify_green_jobs)
```
Comment (ekohl) [inline procedures/foreman/release.md.erb:45]: Depends on https://github.com/theforeman/theforeman-rel-eng/pull/108

File: procedures/foreman/release.md.erb [hunk 2]
```diff
@@ -42,22 +42,18 @@
...
+<% if (is_rc || full_version.end_with?('0')) -%>
+- [ ] Run `make -C locale tx-update` in foreman <%= short_version %>-stable
+<% end -%>
+- [ ] Update release version similar to [here](https://github.com/theforeman/theforeman-rel-eng/commit/2029a9688da00d9c385c3438dd71b594ba5f728e)
+- [ ] Tag the projects
+  - [ ] Create tags [tag_project](https://github.com/theforeman/theforeman-rel-eng/blob/master/tag_project)
+  - [ ] Push tags [tag_project](https://github.com/theforeman/theforeman-rel-eng/blob/master/tag_push)
```
Comment (ekohl) [inline procedures/foreman/release.md.erb:52]: Depends on https://github.com/theforeman/theforeman-rel-eng/pull/109

**Review by ekohl — COMMENTED** [review]
Consider this an RFC. I hope @tbrisker and others will weigh in.

I hope we can eventually use the same scripts for Katello so that the processes are further aligned.

**Comment by ekohl** [comment]
Dependencies have been merged.

**Review by ehelms — APPROVED** [review]

**Pull request merged**